client/src/main/kotlin/io/provenance/client/grpc/AbstractPbClient.kt

```diff
@@ -24,11 +36,14 @@ open class AbstractPbClient<T : ManagedChannelBuilder<T>>(
     open val gasEstimationMethod: GasEstimator,
     open val fromAddress: (String, Int) -> T,
     channel: ManagedChannel,
+    txPollingThreadPool: ExecutorService = Executors.newFixedThreadPool(5)
```

Comment:
A threadpool for dealing with polling tx status... any input on its creation here?